assets/src/dashboard/style.css

```diff
@@ -20,9 +20,22 @@
  * Hides all controls that interfere with the dashboard UI.
  */
 
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+html {
```

Comment:
This might be too intrusive. I want to hear what @swissspidy thinks about this. But generally, modifying `:root` font-size and using `rem` measurements are a big uncertain to me, both from the point of view of ergonomics and correctness.

Reply:
Yeah these changes not only affect the Dashboard itself, but also the rest of WordPress admin, changing the default font family and font size for lots of things in it.

Before:

<img width="1087" alt="Screenshot 2020-03-20 at 18 24 02" src="https://user-images.githubusercontent.com/841956/77189646-2f768c80-6ad8-11ea-8d79-e352174368de.png">

After:

<img width="1086" alt="Screenshot 2020-03-20 at 18 23 55" src="https://user-images.githubusercontent.com/841956/77189658-330a1380-6ad8-11ea-89fa-188205b4b523.png">

For example, compare the font family (Apple System font vs. Google Sans) in the admin menu, or the font size of the "Collapse menu" button in the admin menu (13px vs 14px).

If there is now way to use rem's only for the Dashboard, I'd probably lean towards avoiding it in order to not cause conflicts.